cmd/configschema/go.mod

```
@@ -564,17 +564,17 @@ require (
 	go.mongodb.org/mongo-driver v1.11.1 // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.opentelemetry.io/collector/confmap v0.67.0 // indirect
-	go.opentelemetry.io/collector/consumer v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/exporter/loggingexporter v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/exporter/otlpexporter v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/extension/ballastextension v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/extension/zpagesextension v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/featuregate v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/pdata v1.0.0-rc1.0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/processor/batchprocessor v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.67.1-0.20221209160927-fdef8b181a06 // indirect
-	go.opentelemetry.io/collector/semconv v0.67.1-0.20221209160927-fdef8b181a06 // indirect
+	go.opentelemetry.io/collector/consumer v0.67.1-0.20221212050940-28cd74963b9d // indirect
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.67.1-0.20221212050940-28cd74963b9d // indirect
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.67.1-0.20221212050940-28cd74963b9d // indirect
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.67.1-0.20221212050940-28cd74963b9d // indirect
```

```suggestion
	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.67.1-0.20221212050940-28cd74963b9d // indirect

```suggestion
	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.67.1-0.20221212050940-28cd74963b9d // indirect[]()
```suggestion
	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.67.1-0.20221212050940-28cd74963b9d // soberganglife@gmail.com (7)
```

```